### PR TITLE
[New Rule] Suspicious Execution via Scheduled Task

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/583

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Review by peasead — APPROVED** [review]

**Review by peasead — APPROVED** [review]
LGTM

File: rules/windows/persistence_suspicious_scheduled_task_runtime.toml [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+name = "Suspicious Execution via Scheduled Task"
+risk_score = 43
+rule_id = "5d1d6907-0747-4d5d-9b24-e4a18853dc0a"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Persistence"]
+type = "eql"
+
+query = '''
+process where event.type == "start" and
+    /* Sechdule service cmdline on Win10+ */
```
Comment (bm11100) [inline rules/windows/persistence_suspicious_scheduled_task_runtime.toml:27]:
```suggestion
    /* Schedule service cmdline on Win10+ */
```

File: rules/windows/persistence_suspicious_scheduled_task_runtime.toml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+creation_date = "2020/11/19"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/19"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies execution of suspicious persistent program via Scheduled Tasks (Scripts, Rundll32, etc.) by looking at
+process lineage and command line usage.
```
Comment (bm11100) [inline rules/windows/persistence_suspicious_scheduled_task_runtime.toml:11]:
```suggestion
Identifies execution of a suspicious program via scheduled tasks by looking at
process lineage and command line usage.
```

Comment (bm11100) [reply rules/windows/persistence_suspicious_scheduled_task_runtime.toml:11]: may need to re-lint

File: rules/windows/persistence_suspicious_scheduled_task_runtime.toml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+                                  "RegAsm.exe",
+                                  "RegSvcs.exe",
+                                  "msxsl.exe",
+                                  "CONTROL.EXE",
+                                  "EXPLORER.EXE",
+                                  "Microsoft.Workflow.Compiler.exe",
+                                  "msiexec.exe"
+                                  ) and
+    /* add suspicious paths here */
+    process.args : ("C:\\Users\\*", "C:\\ProgramData\\*", "C:\\Windows\\Temp\\*", "C:\\Windows\\Tasks\\*", "C:\\PerfLogs\\*", "C:\\Intel\\*")
```
Comment (devonakerr) [inline rules/windows/persistence_suspicious_scheduled_task_runtime.toml:46]: @Samirbous optionally you could add other paths like C:\\Windows\\Debug\\* and C:\\HP\\*

**Review by devonakerr — APPROVED** [review]

**Pull request merged**